components/meta-packages/history/history

```diff
@@ -880,6 +880,9 @@ library/python/python-gnome-extras-27@2.25.3,5.11-2018.0.0.5
 library/python/python-gnome-extras@2.25.3,5.11-2018.0.0.5
 library/python-2/argparse@1.2.1,5.11-2016.0.1.0
 library/python-2/argparse-26@1.2.1,5.11-2016.0.1.0
+library/python-2/coherence-26@0.6.6.2-2016.0.1.2
```

Comment:
The latest one I see is `FMRI: pkg://openindiana.org/library/python-2/coherence-26@0.6.6.2-2016.0.1.2:20180211T133129Z`, should this be **2016.0.1.3**?

Reply:
No, as we just move definition from coherence history file to global history file (the package is already obsolete).